pom.xml

```diff
@@ -315,6 +322,20 @@ under the License.
                 <version>1.12.10</version>
             </dependency>
 
+            <!-- For dependency convergence on crate-jdbc mismatch -->
+            <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>31.1-jre</version>
```

Comment:
Why we are adding guava dependency here ? Looks like start of dependency hell.

Reply:
Because convergence, it will fail CI if not added